ui/ducks/index.js

```diff
@@ -16,6 +17,7 @@ export default combineReducers({
   activeTab: (s) => (s === undefined ? null : s),
   metamask: metamaskReducer,
   appState: appStateReducer,
+  ENS: ensReducer,
```

Comment:
Not of any real consequence but can this be `ens`?

Reply:
not a hill i'd die on but this slice will be going away in favor of using a background controller in the future so i'd just say let this slide for now until I can remove this piece of code.